### PR TITLE
[Robot][RD_mappings] New negative test for user without RD2 enabled should not see the settings option for field mappings

### PR DESCRIPTION
**Pull request by rjanjanam**
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: robot/Cumulus/resources/NPSPSettingsPageObject.py [hunk 1]
```diff
@@ -49,6 +49,15 @@ def open_sub_link(self,title):
...
         self.selenium.capture_page_screenshot()
 
+    @capture_screenshot_on_error
+    def verify_sub_link_present(self, title, bool):
+        """ Asserts if the sublink is present or not present
+            eg: Verify Sub Link Present     <Linkoption>   True/False
+        """
+        locator=npsp_lex_locators['npsp_settings']['panel_sub_link'].format(title)
+        assert bool == str(self.npsp.check_if_element_exists(locator)), "Expected value to be {} but found {}".format(
+            bool, self.npsp.check_if_element_exists(locator)
```
Comment (skristem) [inline robot/Cumulus/resources/NPSPSettingsPageObject.py:59]: curious on why are you trying to print the keyword used as the error message when this fails

File: robot/Cumulus/tests/browser/recurring_donations/user_without_rd2_should_not_see_status_mappings.robot [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    [Documentation]               On an org without rd2 enabled, a user
+     ...                          Should not see the option Status to state mapping option
+     ...                          Under NPSP Settings
+
+
+    [tags]                       unstable                      W-8211276
+
+    Go To Page                   Custom                        NPSP_Settings
+    Open Main Menu               Recurring Donations
+    Verify Sub Link Present      Status to State Mapping       False
```
Comment (skristem) [inline robot/Cumulus/tests/browser/recurring_donations/user_without_rd2_should_not_see_status_mappings.robot:23]: this test case looks very short, cant this be combined with any other test case?

Comment (rjanjanam) [reply robot/Cumulus/tests/browser/recurring_donations/user_without_rd2_should_not_see_status_mappings.robot:23]: I know, But that is how we decided to create and pointed the workitem accordingly. This is the beginning for a series of field mapping automation stories to follow.

Comment (skristem) [reply robot/Cumulus/tests/browser/recurring_donations/user_without_rd2_should_not_see_status_mappings.robot:23]: so will u be adding more to this test case later on in future sprints?

Comment (rjanjanam) [reply robot/Cumulus/tests/browser/recurring_donations/user_without_rd2_should_not_see_status_mappings.robot:23]: This test case is a valid scenario and is as is part of the validation as we do not want this sub menu option to be present when user has no RD2 enabled. The following ones are for scenarios where rd2 is enabled and user sees the submenu options and the following actions.

**Review by rhakeshj — APPROVED** [review]

File: robot/Cumulus/resources/NPSPSettingsPageObject.py (hunk 1 above)
Comment (rjanjanam) [reply robot/Cumulus/resources/NPSPSettingsPageObject.py]: Updated the message to be meaningful.

**Review by skristem — APPROVED** [review]

**Pull request merged**